miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultSubprojectService.java

```diff
@@ -26,7 +26,7 @@
   protected static final Logger log = LoggerFactory.getLogger(DefaultSubprojectService.class);
 
   @Autowired
-  private SQLProjectDAO sqlProjectDAO;
+  private HibernateProjectDao sqlProjectDAO;
```

Comment:
It's everywhere, because if it's named anything else, the autowiring would  have failed.